go/vt/vtgate/vindexes/vschema.go

```diff
@@ -965,33 +1055,6 @@ func (vschema *VSchema) FindView(keyspace, name string) sqlparser.SelectStatemen
 	return statement
 }
 
-func (vschema *VSchema) findKeyspaceAndTableBySource(source *Source) (*Keyspace, *Table, error) {
```

Comment:
Was used by `buildKeyspaceReferences` to verify that a reference `source` can be resolved, but now this is replaced by a call to `(*VSchema).FindTable` plus some error checking.